### PR TITLE
Add -Inf case

### PR DESCRIPTION
**Pull request by uri-99**
This PR adds execution_price: -Infinity case
This PR also cleans and adds this to the .py parser

### CODE REVIEW

**Review by dubzn — APPROVED** [review]

File: scripts/utility/swap_expected_result.txt
Comment (dubzn) [inline scripts/utility/swap_expected_result.txt:1]: remove this please!

File: scripts/utility/uni_values.txt
Comment (dubzn) [inline scripts/utility/uni_values.txt:1]: remove this please :pray:

File: scripts/utility/parse_to_cairo_struct.py [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+#     amount_1_before: 1994009290088178439,
+#     amount_1_delta: IntegerTrait::<i256>::new(662011820624678025, true),
+#     execution_price: 52449980110816002175186131547,
+#     fee_growth_global_0_X128_delta: 510423550381407695195061911147652317,
+#     fee_growth_global_1_X128_delta: 0,
+#     pool_price_after: 52765855989621530048506654453,
+#     pool_price_before: 79228162514264337593543950336,
+#     tick_after: IntegerTrait::<i32>::new(8130, true),
+#     tick_before: IntegerTrait::<i32>::new(0, false),
+# },
```
Comment (dubzn) [inline scripts/utility/parse_to_cairo_struct.py:30]: why are we changing this in this PR? :thinking:

**Review by dubzn — CHANGES_REQUESTED** [review]

File: scripts/utility/uni_values.txt
Comment (rcatalan98) [reply scripts/utility/uni_values.txt:1]: Should we add the `.txt` to the `gitignore` file? So we can avoid this happening in the future

File: crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo [hunk 2]
```diff
@@ -2028,6 +2028,8 @@ mod YASPoolTests {
         if token_0_swapped_amount == 0
             && token_1_swapped_amount == 0 { //this avoids 0/0 , no tokens swapped = exec_price: 0
             0
+        } else if token_0_swapped_amount == 0 {
+            '-Infinity'.into()
```
Comment (rcatalan98) [inline crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo:2032]: Can you add a comment describing how this works? Something similar to what you mentioned in Slack.
Describing the different cases and the output for each one

**Review by rcatalan98 — APPROVED** [review]

**Pull request merged**